bigtable/google/cloud/bigtable/instance.py

```diff
@@ -481,3 +483,110 @@ def list_app_profiles(self):
         """
         resp = self._client._instance_admin_client.list_app_profiles(self.name)
         return [AppProfile.from_pb(app_profile, self) for app_profile in resp]
+
+    def get_iam_policy(self):
+        """
+        Gets the access control policy for an instance resource.
+
+        Example:
+            >>> from google.cloud.bigtable.client import Client
+            >>>
+            >>> from google.cloud.iam import Policy
+            >>>
+            >>> client = Client(admin=True)
+            >>>
+            >>> instance = client.instance('[INSTANCE_ID]')
+            >>>
+            >>> policy_latest = instance.get_iam_policy()
+            >>>
+            >>> print (policy_latest.owners)
+
+        Returns:
+            A :class:`~google.cloud.iam.Policy` instance.
+        """
+        instance_admin_client = self._client._instance_admin_client
+        resp = instance_admin_client.get_iam_policy(resource=self.name)
+        return Policy.from_api_repr(self._to_dict_from_policy_pb(resp))
+
+    def set_iam_policy(self, policy):
+        """
+        Sets the access control policy on an instance resource. Replaces any
+        existing policy.
+
+        Example:
+            >>> from google.cloud.bigtable.client import Client
+            >>>
+            >>> from google.cloud.iam import Policy
+            >>>
+            >>> client = Client(admin=True)
+            >>>
+            >>> instance = client.instance('[INSTANCE_ID]')
+            >>>
+            >>> policy = instance.get_iam_policy()
+            >>>
+            >>> policy.owners = [Policy.user("test_iam@test.com"),
+            ...                Policy.service_account("sv_account@gmail.com")]
+            >>>
+            >>> instance.set_iam_policy(policy)
+            >>>
+            >>> policy_latest = instance.get_iam_policy()
+            >>>
+            >>> print (policy_latest.owners)
```

Comment:
I know other docstrings have it, but we *don't* need the extra `>>>` lines between each statement. 

I would argue that using the "interpreter prompt" bits for example syntax is actually harmful:  it makes the example harder to copy-and-paste for users who are exploring the library.  Instead, the examples could be set off with `.. code-block:: python`, e.g.:


```python
        Example:

        .. code-block:: python

            from google.cloud.bigtable.client import Client
            from google.cloud.iam import Policy
            
            client = Client(admin=True)
            instance = client.instance('[INSTANCE_ID]')
            policy = instance.get_iam_policy()
            policy.owners = [
                Policy.user("test_iam@test.com"),
                Policy.service_account("sv_account@gmail.com"),
            ]
            instance.set_iam_policy(policy)
            policy_latest = instance.get_iam_policy()
            print (policy_latest.owners)
```